librosa/core/audio.py

```diff
@@ -481,23 +481,31 @@ def resample(
         target sampling rate
 
     res_type : str
-        resample type (see note)
+        resample type
+
+        'kaiser_best' (default)
+            `resampy` high-quality mode
+        'kaiser_fast'
+            `resampy` faster method
+        'fft' or 'scipy'
+            `scipy.signal.resample` Fourier method. (fast)
```

Comment:
This one should be marked slow.

Reply:
Is it really slow? I ran some benchmark, it was fast.
https://colab.research.google.com/drive/1XgSOvWlRIau1FYwQG_yRSAhDK3KB8bEL?usp=sharing

Library                  | Time on CPU (ms)
------------------------ | ----------------
soxr (HQ)                | 7.2
scipy.signal.resample    | 13.4
soxr (VHQ)               | 15.8
torchaudio               | 19.2
lilfilter                | 21.4
julius                   | 23.1
resampy (kaiser_fast)    | 62.6
samplerate (sinc_medium) | 92.5
resampy (kaiser_best)    | 256
samplerate (sinc_best)   | 397

Reply:
It's fast when the input length is amenable to FFT (ie n is highly composite), but it can be *very* slow in some cases, eg when n has few (large) prime factors.

Reply:
ah, this seems to have improved a bit since i last benchmarked it!  perhaps strike my remarks from the record. :)

Reply:
Well, I found this sentence on scipy docs. 
>As noted, resample uses FFT transformations, which can be very slow if the number of input or output samples is large and prime; see scipy.fft.fft.

Reply:
I'll just remove (fast) or (slow) from `scipy.signal.resample` description.

Reply:
Hah, fair enough.  I dug up my 2016 scipy lightning talk about this, and the difference back then on `n=400000` (which is still quite composite!) was orders of magnitude.  I just tested locally with n=400009 (next prime up) and it's about parity with everything else.

I dug into the history of scipy revisions, but couldn't quite determine what change was responsible for the improvement.  They did switch the preferred FFT backend at some point, so maybe we now have one with a plan that works well on prime/near-prime lengths?